docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
       AWS_TRANSLATE_ACCESS_KEY: $AWS_TRANSLATE_ACCESS_KEY
       AWS_TRANSLATE_SECRET_KEY: $AWS_TRANSLATE_SECRET_KEY
       AWS_TRANSLATE_REGION: $AWS_TRANSLATE_REGION
+      DEBUG_INTERNAL_IP: $DEBUG_INTERNAL_IP
```

Comment:
I will recommend we only use `DEBUG` mode in local development. cc: @batpad

Reply:
Agree. @GregoryHorvath do you think it's fine to make that change as part of this PR?